modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/annotatedslice/v0.1.0.json

```diff
@@ -0,0 +1,66 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/bbp/core/entity/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpentsh": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/",
+      "bbpqsh": "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+      "bbppsh": "{{base}}/schemas/bbp/core/person/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/annotatedslice/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/entity/v0.1.0",
+    "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0"
+  ],
+  "shapes": 
+    [
+      {
+        "@id": "this:AnnotatedSliceShape",
+        "@type": "sh:NodeShape",
+        "label": "Annotated slice shape definition",
+        "targetClass": "nsg:AnnotatedSlice",
+        "nodeKind": "sh:BlankNodeOrIRI",
+        "and": [
+          {
+            "node": "bbpentsh:EntityShape"
+          },
+          {
+            "property" : [ {
+              "path" : "nsg:AnnotationAngle",
+              "name" : "Annotation angle",
+              "description" : "Angle at which the slice is as defined during the annotation process",
+              "node": "bbpquantValue:QuantitativeValueShape",
+              "maxCount" : 1
+            }, {
+              "path" : "nsg:AnnotatorComment",
+              "name" : "Annotator comment",
+              "description" : "Comments added by the annotator during the annotation process",
+              "datatype":"xsd:string",
+              "maxCount" : 1
+            },
+              {
+                "path": "dcterms:hasPart",
+                "name": "Patched cell collection",
+                "description": "The collections of labeled cells",
+                "class": "nsg:LabeledCellCollection",
+                "minCount": 1
+              },
+              {
+                "path": "prov:wasRevisionOf",
+                "name": "fixedStainedSlice",
+                "description": "The fixedStainedSlice this annotatedSlice is a revision of",
```

Comment:
I would suggest to use plain sentence in descriptions like:
"The fixed stained slice this annotated slice is a revision of."

Reply:
As discussed we agreed on the following description:
    "description": "The FixedStainedSlice this AnnotatedSlice is a revision of.",